tests/test_gocamgen.py

```diff
@@ -59,3 +59,18 @@ def test_get_with_froms():
     ca_set.find_or_create_collapsed_association(go_assoc, with_from=header1)
     ca_set.find_or_create_collapsed_association(go_assoc, with_from=header2)
     assert len(ca_set.collapsed_associations) == 1 and ca_set.collapsed_associations[0].with_froms == ["FAKE:12345", "MGI:MGI:1915834"]
+
+
+def test_ref_picker():
+    test_refs = [
+        "GO_REF:0000483",
+        "doi:485930",
+        "WB_REF:WBPaper00003384",
+        "PMID:9834189",
```

Comment:
Would it be possible to flip the PMID and WB_REF?

Reply:
Like, switch in the test to just ensure the `PMID` isn't picked because it's last?
```
        "GO_REF:0000483",
        "doi:485930",
        "PMID:9834189",
        "WB_REF:WBPaper00003384",
```

Reply:
Yeah, sorry, I just noticed that your tests would pass if the algorithm of "pick last" and that made me feel weird. Personal problem, but I couldn't get the idea unstuck from my mind.

Reply:
Good point! I added that flip case to help make it clear.